### ISSUE TITLE
Uncaught TypeError: marker.getBufferRange is not a function

### ISSUE THREAD

**Issue by maddie927** [nwolverson/atom-ide-purescript#85]
1. Open a PS file
2. press alt+enter anywhere

**Atom Version**: 1.8.0
**System**: Mac OS X 10.11.5
**Thrown From**: [ide-purescript](https://github.com/nwolverson/atom-ide-purescript) package, v0.13.5
### Stack Trace

Uncaught TypeError: marker.getBufferRange is not a function

```
At /Users/trotter/.atom/packages/ide-purescript/lib/main.js:8589

TypeError: marker.getBufferRange is not a function
    at /Users/trotter/.atom/packages/ide-purescript/lib/main.js:8589:25
    at /Users/trotter/.atom/packages/ide-purescript/lib/main.js:772:18
    at /Users/trotter/.atom/packages/ide-purescript/lib/main.js:772:22
    at __do (/Users/trotter/.atom/packages/ide-purescript/lib/main.js:9748:187)
    at /Users/trotter/.atom/packages/ide-purescript/lib/main.js:772:18
    at __do (/Users/trotter/.atom/packages/ide-purescript/lib/main.js:9813:98)
    at __do (/Users/trotter/.atom/packages/ide-purescript/lib/main.js:10063:112)
    at atom-workspace.<anonymous> (/Users/trotter/.atom/packages/ide-purescript/lib/main.js:828:19)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:580:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:388:22)
    at KeymapManager.module.exports.KeymapManager.terminatePendingState (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:561:14)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:432:21)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:98:36)
    at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:3:61)
```
### Commands

```
  3x -0:50.4.0 vim-mode:move-up (atom-text-editor.editor.vim-mode.normal-mode.is-focused)
     -0:36.6.0 vim-mode:activate-insert-mode (atom-text-editor.editor.vim-mode.normal-mode.is-focused)
     -0:35.7.0 vim-mode:activate-normal-mode (atom-text-editor.editor.vim-mode.is-focused.insert-mode)
     -0:35.1.0 vim-mode:move-down (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
     -0:35 vim-mode:move-up (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
     -0:33.4.0 purescript:show-quick-fixes (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
     -0:10.7.0 vim-mode:move-to-previous-word (atom-text-editor.editor.vim-mode.normal-mode.is-focused)
     -0:09.0 vim-mode:change-to-last-character-of-line (atom-text-editor.editor.vim-mode.normal-mode.is-focused)
     -0:07.0 autocomplete-plus:confirm (atom-text-editor.editor.vim-mode.is-focused.insert-mode.autocomplete-active)
     -0:04.8.0 vim-mode:activate-normal-mode (atom-text-editor.editor.vim-mode.is-focused.insert-mode)
     -0:02.9.0 vim-mode:move-to-previous-word (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
     -0:02.6.0 vim-mode:activate-characterwise-visual-mode (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
     -0:02.6.0 vim-mode:move-to-end-of-word (atom-text-editor.editor.vim-mode.is-focused.visual-mode)
     -0:02.1.0 vim-mode:repeat-find (atom-text-editor.editor.vim-mode.is-focused.visual-mode)
     -0:01.0 purescript:show-quick-fixes (atom-text-editor.editor.vim-mode.is-focused.visual-mode)
     -0:00.2.0 vim-mode:activate-normal-mode (atom-text-editor.editor.vim-mode.is-focused.visual-mode)
```
### Config

``` json
{
  "core": {
    "disabledPackages": [
      "atom-pair",
      "atom-yeoman",
      "autocomplete-emojis",
      "gist-it",
      "linter-flow",
      "linter-liquid",
      "nerd-treeview",
      "open-this",
      "sort-lines",
      "welcome",
      "build",
      "atom-typescript",
      "term3",
      "linter-elm-make",
      "linter-tslint",
      "timecop"
    ],
    "projectHome": "/Users/trotter/dev/atom",
    "themes": [
      "isotope-ui",
      "base16-tomorrow-dark-theme"
    ]
  },
  "ide-purescript": {}
}
```
### Installed Packages

``` coffee
# User
auto-update-packages, v1.0.1
cursor-history, v0.5.9
editorconfig, v1.4.1
emmet, v2.4.3
file-icons, v1.7.15
github-issues, v0.7.1
haskell-unicode-snippets, v0.1.1
ide-purescript, v0.13.5
isotope-ui, v2.7.1
language-babel, v2.28.0
language-cshtml, v0.2.1
language-dustjs, v0.2.0
language-elm, v1.5.0
language-fsharp, v0.8.4
language-graphql, v0.2.0
language-haskell, v1.7.17
language-purescript, v0.7.0
language-viml, v1.0.0
latex-completions, v0.3.1
linter, v1.11.14
linter-csslint, v1.3.2
linter-eslint, v7.2.4
linter-hlint, v0.4.3
minimap, v4.24.6
pigments, v0.29.2
ssh-config, v0.10.1
symbols-tree-view, v0.13.2
sync-settings, v0.7.2
vim-mode, v0.65.1

# Dev
No dev packages
```

**Comment by born2defy** [comment]
Getting that too.

**Comment by nwolverson** [comment]
Changes in linter, need to change `getBufferRange` to `getRange` as it is now a simple `Marker` not a `TextEditorMarker` or `DisplayMarker` or whatever. Easiest to change after merging #82

**Comment by nwolverson** [comment]
Should now be working (released 0.13.6), can you check

Thanks for reporting!

**Comment by born2defy** [comment]
Just checked and it is working for me.  Great job!

Adam
